src/main/docker/non_managementportal/docker-compose.yml

```diff
@@ -21,8 +21,6 @@ services:
     networks:
       - db
       - default
-    ports:
-      - "5432:5432"
```

Comment:
probably needed if need to access from outside docker (as the top comment suggests)?

Reply:
It is, but this is also defined in the file: src/main/docker/postgres.yml

So it's a question of where do we define it. My docker complained that the port was already assigned, which is why I removed one of the definitions. Since this port is so standard  figured it could be in the postgres config rather than in the docker_compose.

Do you suggest moving it here?